packages/framework/use-slot/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@fluentui-react-native/use-slot",
+  "version": "0.1.1",
+  "description": "Hook function to use a component as a pluggable slot",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/ui-fabric-react-native"
+  },
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "bundle": "fluentui-scripts bundle",
+    "clean": "fluentui-scripts clean",
+    "code-style": "fluentui-scripts code-style",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "start": "fluentui-scripts dev",
+    "start-test": "fluentui-scripts jest-watch",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@fluentui-react-native/merge-props": ">=0.3.2 <1.0.0"
+  },
+  "devDependencies": {
+    "@uifabricshared/build-native": "^0.1.1",
+    "react": "16.13.1",
+    "react-native": "^0.63.4"
+  },
+  "peerDependencies": {
+    "react": ">=16.13.1",
+    "react-native": ">0.60.0"
```

Comment:
Curious why this is much lower than the version used in devDependencies?

Reply:
This was copied from use-slots so I didn't really reassess this. What this is saying is effectively 'when you bundle me, you need to bundle a version of react above X and a version of react-native above Y.' This particular package would work with 0.60 and beyond, but we should figure out what the min-supported version for the library really is and make sure all the declared peer dependencies are consistent. Likely dep-check can help with that but it's a good follow-up.

Reply:
I think we implicitly decided RN 0.63 was the minimum supported version when we updated to RN 0.63 and changed all of our dependencies to reflect that.

Reply:
Turns out I copied use-slots to start this package which was one of only three packages that didn't have the correct peer dependencies set. I updated this package and the other ones in the project that were incorrect as well.